101-machine-learning-linkedserivce-create/azuredeploy.json

```diff
@@ -0,0 +1,125 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "amlWorkspaceName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the AML workspace"
+            }
+        },
+        "linkName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the LinkedService"
+            }
+        },
+        "synapseWorkspaceResourceId": {
+            "type": "string",
+            "metadata": {
+                "description": "ResourceId of the Synapse workspace which you want to link to the aml workspace."
+            }
+        },
+        "sparkPools": {
+            "type": "array",
+            "defaultValue": [
+                {
+                    "computeName": "compute1",
+                    "poolName": "sparkPool1"
+                },
+                {
+                    "computeName": "compute2",
+                    "poolName": "sparkPool2"
+                }
+            ],
+            "metadata": {
+                "description": "Spark pools you want to attach to the aml workspace. Pool name is the name of spark pool shown in Synapse workspace, while compute name is the alias for this pool after it is attached to aml workspace."
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",

```

Comment:
remove the defaultValue since you require a specific location

Reply:
we're not require a specific location, will keep the defaultValue and remove that from the parameters file instead.